overlord/snapstate/devicectx.go

```diff
@@ -29,6 +29,10 @@ import (
 type DeviceContext interface {
 	// Model returns the governing device model assertion for the context.
 	Model() *asserts.Model
+
+	// OldModel returns the old model assertion for the context. This will only return anything in a remodel context.
```

Comment:
s/for the context/for the device/   s/anything/something/  s/in a remodel context/for a remodel context/